client/utils/gitUtils.js

```diff
@@ -2,7 +2,7 @@ const lc = (pattern, string) => string.replace(pattern, pattern.toLowerCase());
 const timeZone = 'UTC';
 const options = { month: 'short' };
 
-export const gitUpdatedDateToString = (dateString, locale = 'default') => {
+export const gitUpdatedDateToString = (dateString, locale = 'en-US') => {
```

Comment:
I don't think this change should be made. It would be forcing users from different regions to view dates in formats they don't expect. Any reason for this change?

Reply:
First, thanks!
That change was made because the test kept failing. The formatting of the return value was different from the formatting of the expted value after running the test. I don't know why that was happening, thought. We summized that it is because I am located in Jamaica outside of the US.

Reply:
Okay got it!

> We summized that it is because I am located in Jamaica outside of the US.

I still don't think this change should stick though. I forced my locale to `en-JM` and found it's the same as `en-US`, based on the expected format of the function. So maybe your device's locale could still be different somehow?

What I did notice though is that the tests at [gitUtils.test.js#L4:L8](https://github.com/w3c/aria-at-app/blob/19298fd510b35177db4bad5772cec0f21687e1b4/client/utils/gitUtils.test.js#L4:L8) and [gitUtils.test.js#L10:L14](https://github.com/w3c/aria-at-app/blob/19298fd510b35177db4bad5772cec0f21687e1b4/client/utils/gitUtils.test.js#L10:L14) are expecting a specific format by default (which seems to typically be `en-` related locales) by it not explicitly providing the locale on lines 6 & 12 to the `gitUpdatedDateToString` function. So this could cause an issue when running the tests on a device not using that format.

So I think you could try reverting the change to the `locale`'s default value and update those 2 tests to explicitly use `en-US` and see how that runs?

Reply:
Ok. I'll try it out.

Reply:
That work, @howard-e!